### PR TITLE
Reduce doc sizes to make kairos-agent smaller

### PR DESCRIPTION
**Pull request by mauromorales**
In total I manage to reduce the size from 15M to 8.1M (requires to run the cleanup script). If this is still too much for the binary we can stop including them, but it's still useful to do these fixes for maintenance and Google speed test

Additional:

- fix issue with descriptions including the entire document
- remove Google ID (since it was a sample ID but it was still loading the JS)

### CODE REVIEW

File: hugo.toml [hunk 1]
```diff
@@ -105,9 +106,6 @@ github_branch = "main"
 # Google Custom Search Engine ID. Remove or comment out to disable search.
 #gcs_engine_id = "011737558837375720776:fsdu1nryfng"
 
-# Enable Algolia DocSearch
-algolia_docsearch = false
```
Comment (jimmykarily) [inline hugo.toml:109]: this is not what we are using for search right?

File: scripts/embedded-docs-cleanup.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```
Comment (jimmykarily) [inline scripts/embedded-docs-cleanup.sh:1]: who runs this script? It's in the kairos-agent side?

File: scripts/embedded-docs-cleanup.sh [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+location=${1:-public}
+
+find "$location" -type f -name "*.xml" -exec test -e {} \; -exec rm -f {} +
+find "$location" -type f -name "swagger-ui*" -exec test -e {} \; -exec rm -f {} +
+rm -rf "$location"/blog
```
Comment (jimmykarily) [inline scripts/embedded-docs-cleanup.sh:6]: does this mean we get 404s when a blog link is clicked?

**Review by jimmykarily — APPROVED** [review]
We don't save much space but if it works, let's do it.

File: hugo.toml (hunk 1 above)
Comment (mauromorales) [reply hugo.toml:109]: nope

File: scripts/embedded-docs-cleanup.sh (hunk 2 above)
Comment (mauromorales) [reply scripts/embedded-docs-cleanup.sh:1]: will need to be run in the package of the kairos-agent where we embed the docs

File: scripts/embedded-docs-cleanup.sh (hunk 3 above)
Comment (mauromorales) [reply scripts/embedded-docs-cleanup.sh:6]: ah this was just an experiment, good catch thanks!

**Pull request merged**